biocrnpyler/parameter.py

```diff
@@ -30,21 +30,24 @@
 
 #### Initial Conditions are also Parameters
 The initial condition of any Species (or Component) will also be looked up as a parameters automatically.
-    Initial conditions can be customized in through the custom_initial_condition keyword in the Mixture constructor.
+    Initial conditions can be customized in through the custom_initial_conditions keyword in the Mixture constructor.
     custom_initial_conditions will take precedent to parameter initial conditions.
 
     During compilation, Mixture.set_initial_condition() checks for parameters for all species in the following order:
 
-    # First checks if (mixture.name, repr(species) is in the self.custom_initial_condition_dict
-    # Then checks if (repr(species) is in the self.custom_initial_condition_dict
+    # First checks if (mixture.name, repr(species)) is in the self.custom_initial_condition_dict
+    # Then checks if (repr(species)) is in the self.custom_initial_condition_dict
     # Then checks if (mixture.name, component.name) is in the self.custom_initial_condition_dictionary
     # Then checks if (component.name) is in the self.custom_initial_condition_dictionary
 
-    # Then checks if (mixture.name, repr(species) is in the parameter dictionary
+    # Then checks if (mixture.name, repr(species)) is in the parameter dictionary
     # Then checks if repr(species) is in the parameter dictionary
-    # Then checks if (mixture.name, component.name) is in the parameter dictionary
+    # Then checks if (mixture.name, component.name)) is in the parameter dictionary
     # Then checks if component.name is in the parameter dictionary
     # Then defaults to 0
```

Comment:
Probably my bad leaving these comments in Parameter, but can you just delete them all? this information is no longer valid and details are given in the Mixture class set_initial_condition function.